content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1132 @@
+
+In previous versions of Bevy scenes were spawned like this:
+
+```rust
+commands.spawn_scene(asset_server.load("some.gltf#Scene0"));
+```
+
+This worked, but it made it difficult to actually use the scene in practice. Repositioning the scene required either manually grabbing the scene's entity id and then repositioning it when it spawns, or creating a parent entity with the relevant transform components and "spawning the scene as a child". Additionally, it was challenging to extend scenes with new components.
+
+In **Bevy 0.8**, we've added a [`SceneBundle`], which brings scene spawning in line with our other entity construction APIs:
+
+```rust
+commands.spawn_bundle(SceneBundle {
+    scene: asset_server.load("some.gltf#Scene0"),
+    ..default()
+})
+```
+
+When the scene loads, it will automatically spawn underneath the entity created by the `spawn_bundle` command. This makes it much easier to transform the scene and add new components at the "root":
+
+```rust
+commands
+    .spawn_bundle(SceneBundle {
+        scene: asset_server.load("player.gltf#Scene0"),
+        transform: Transform::from_xyz(10.0, 20.0, 30.0),
+        ..default()
+    })
+    .insert(Player::default());
+```
+
+[`SceneBundle`]: https://docs.rs/bevy/0.8.0/bevy/scene/struct.SceneBundle.html
+
+## Parallel Frustum Culling
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+Bevy uses ["frustum culling"](/news/bevy-0-6/#visibility-and-frustum-culling) to skip drawing entities that are outside of the camera's view. In **Bevy 0.8**, frustum culling is now done in parallel. When culling thousands of entities, this yields significant performance improvements:
+
+### frustum culling system time vs number of entities culled (lower is better) 
+
+![parallel frustum culling](parallel_frustum_culling.png)
+
+Note that "parallel b" stand for "parallel batch size" (number of entities in each batch). We selected 1024 as our batch size in **Bevy 0.8**, as that performs better.
+
+## Automatic Mesh Tangent Generation
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump), @jakobhellermann, @DJMcNab</div>
+
+Vertex tangents are used in tandem with normal maps to give meshes more detailed normals when rendering them. Some imported meshes have normal maps, but don't have vertex tangents calculated. Bevy can now automatically generate vertex tangents for [`Meshes`][`Mesh`] that are missing them using the defacto industry-standard MikkTSpace library / algorithm (Godot, Unity, Unreal, and Blender all use this).
+
+We have started maintaining [our own fork](https://github.com/bevyengine/bevy/tree/v0.8.0/crates/bevy_mikktspace) of the [gltf-rs/mikktspace crate](https://github.com/gltf-rs/mikktspace) so we can:
+1) update dependencies at the speed required for Bevy;
+2) [start reining in the unsafe code](https://github.com/bevyengine/bevy/pull/4932) (it currently uses unsafe Rust code auto-generated from the original `mikktspace.h` written in C).
```

Comment:
this renders with just "C)." on its own line
<img width="917" alt="Screenshot 2022-07-29 at 01 24 35" src="https://user-images.githubusercontent.com/8672791/181653406-1d01e2f5-7ed7-4bf8-acd1-ac1444e6d8d7.png">

Reply:
That looks like an unfortunate line break to me? The rest of the text is there above.

Reply:
Just rephrased it to remove parens / avoid the unfortunate confusion

Reply:
yup my issue was with the linebreak, not so much with the formulation. it didn't change much, now with "in C." in its own line